frontend/src/pages/Onramper/index.js

```diff
@@ -0,0 +1,22 @@
+import { useSearchParams } from 'react-router-dom';
+import "./index.css";
+
+export const Onramper = () => {
```

Comment:
@kon-rad I saw that we are able to post messages directly to the iframe. Depending on how fast everything loads, this may be something to keep in mind for reducing the load time. For example

Option A) Load the iframe. Fetch and compile the new url from useSearchParams. Update the iframe src to trigger reloading it
Option B) Load the iframe, fetch and compile the new url from useSearchParams. Update the iframe using post-message. See https://docs.onramper.com/docs/widget-theme